provider/source/src/time_zones/convert.rs

```diff
@@ -49,32 +51,29 @@ impl DataProvider<TimeZoneFormatsV1Marker> for SourceDataProvider {
             .time_zone_names;
 
         fn parse_hour_format(hour_format: &str) -> (Cow<'static, str>, Cow<'static, str>) {
-            // e.g. "+HH:mm;-HH:mm" -> ("+HH:mm", "-HH:mm")
+            // e.g. "+HH:mm;-hh:mm" -> ("+H:m", "-h:m")
```

Comment:
Thought: It makes sense what you did here, but lowercase "h" generally means 12-hour time, but that doesn't make sense for GMT offset formatting. This mapping you chose seems okay, though nonstandard.

Reply:
`Hhm` are the only characters I'm sure are not used by patterns, so I went with this.

Reply:
It would be slightly cleaner to use `MultiNamedPlaceholderPattern` for these, although it's slightly more data.